components/base/Typography.tsx

```diff
@@ -136,6 +137,50 @@ export const listStyles = css`
   }
 `;
 
+// Table styles
+export const tableStyles = css`
+  border-spacing: 0;
+
+  thead {
+    th {
+      font-family: ${ROBOTO};
+      font-weight: 500;
+      font-size: 13px;
+      color: ${HEADLINE_TEXT};
+      line-height: 17px;
+      padding: 8px 16px;
+    }
+  }
+
+  tbody {
+    tr {
+      box-shadow: 0 0 0 1px ${GREY_2}; /* https://stackoverflow.com/a/48831092 */
+
+      &:nth-of-type(1) {
+        border-top-left-radius: 4px;
+        border-top-right-radius: 4px;
+      }
+
+      &:last-child {
+        border-bottom-left-radius: 4px;
+        border-bottom-right-radius: 4px;
+      }
+
+      &:nth-of-type(odd) {
+        background: ${GREY_1};
+      }
+    }
+
+    td {
+      font-family: ${ROBOTO};
+      font-size: 14px;
+      line-height: 20px;
+      color: ${PARAGRAPH_TEXT};
+      padding: 10px 16px;
```

Comment:
Can we bump up the vertical padding a bit more? It feels pretty tight compared to the designs